ci-operator/config/openshift/installer/openshift-installer-release-4.14.yaml

```diff
@@ -691,6 +691,18 @@ tests:
         AGENT_USE_ZTP_MANIFESTS=true
     workflow: agent-e2e-generic
   timeout: 6h0m0s
+- as: e2e-agent-sno-ipv4-pxe
+  cluster: build05
+  optional: true
+  run_if_changed: /agent/
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      DEVSCRIPTS_CONFIG: |
+        AGENT_E2E_TEST_SCENARIO=SNO_IPV4
```

Comment:
I'm not fully sure that this is required, the CI bot should update the branch jobs accordingly from the master one